### PR TITLE
feature/login

### PR DESCRIPTION
**Pull request by seffradev**

### CODE REVIEW

**Comment by seffradev** [comment]
woah docker in a feature branch?????

**Comment by seffradev** [comment]
wooo you fixed the issue!!!!!

**Review by seffradev — COMMENTED** [review]
LGTM!

**Review by ABerger81 — CHANGES_REQUESTED** [review]
Input fields in login.html and register.html are "disappearing" or extending out of the right side of their container

File: transcription/main.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+def main():
+    print("Hello, World!")
```
Comment (seffradev) [inline transcription/main.py:2]: Should this really be `Hello, World!`? Isn't that outdated?

File: docker/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+# Python Container: Replace ./python-app with the path to your Python application and main.py with your Python program entry point.
+# Volumes: A persistent volume for MariaDB is defined to ensure data persistence. Ensure volume paths and configurations align with your project structure and storage requirements.
+# Networks: By default, Docker Compose sets up a single network for your app's services to communicate with each other. If you need isolated networks or custom configurations, you can define them under a networks key.
+# Security: This configuration includes default passwords and configurations for simplicity. For production environments, ensure you use secure passwords, consider network configurations, and review each service's security best practices.
+# After creating this docker-compose.yml, you can start all services with docker-compose up and stop them with docker-compose down. Remember, this file is a starting point, and further customization might be necessary to fit your specific project requirements.
+
+version: '3.8'
```
Comment (seffradev) [inline docker/docker-compose.yml:12]: Is this the latest version?

**Review by seffradev — COMMENTED** [review]
uuwaaaaaaaaaah

File: transcription/main.py (hunk 1 above)
Comment (seffradev) [reply transcription/main.py:2]: Discussion...

**Review by ABerger81 — APPROVED** [review]

**Pull request merged**